EMS/client-helper-bundle/src/Resources/config/routing/asset.xml

```diff
@@ -1,11 +1,12 @@
 <?xml version="1.0" encoding="UTF-8" ?>
-
 <routes xmlns="http://symfony.com/schema/routing"
         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-        xsi:schemaLocation="http://symfony.com/schema/routing http://symfony.com/schema/routing/routing-1.0.xsd">
+        xsi:schemaLocation="http://symfony.com/schema/routing
+        https://symfony.com/schema/routing/routing-1.0.xsd">
```

Comment:
```suggestion
        xsi:schemaLocation="https://symfony.com/schema/routing/routing-1.0.xsd">
```

Reply:
I saw it at multiple places

Reply:
Just took the xml example from symfony https://symfony.com/doc/current/routing.html#creating-routes-in-yaml-xml-or-php-files

Reply:
Ok

Reply:
Cleaned it a little bit https://github.com/ems-project/elasticms/pull/221/commits/3db71ea18b72ae20019fe1e6d89b2112ff9db540